server/Dockerfile

```diff
@@ -1,24 +1,22 @@
-FROM registry.access.redhat.com/ubi8-minimal
+FROM quay.io/alfresco/alfresco-base-java:11
```

Comment:
Why are we making this change?  The dockerfile is already including openjdk11.  This kind of change make it difficult to contribute back changes to the project.  It makes it difficult to maintain upgrades. Are we open to now fixing issues that this may cause in other parts of the keycloak platform? To paraphrase a question on the mailing list about this..."why do you need _alfresco's java 11_ , especially in the container?"

Reply:
I think this way we have a greater control over the Java version in order to fix the security vulnerabilities, rather than depending on OS package manager to include the latest version.

@rgauss , can you please clarify?

Reply:
Yes.  As I recall we decided that all Java-based images we produce should ideally come from our alfresco-base-java image.  If that starts to prove difficult to maintain we should re-evaluate that decision.

Reply:
Did we verify that the resulting image has keytool present? https://issues.alfresco.com/jira/browse/AUTH-408?focusedCommentId=642243&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-642243

Reply:
Keytool is present in the alfresco-base-java image but it is not in the path. 
The base image will be updated to add keytool to the path. This is done by [AUTH-456](https://issues.alfresco.com/jira/browse/AUTH-456)